v_3_0_0/master_template.go

```diff
@@ -360,210 +360,143 @@ write_files:
               hostPath:
                 path: /etc/kubernetes/ssl/etcd
 {{ end -}}
-- path: /srv/kubedns-cm.yaml
+- path: /srv/coredns.yaml
   owner: root
   permissions: 0644
   content: |
     apiVersion: v1
-    kind: ConfigMap
+    kind: ServiceAccount
     metadata:
-      name: kube-dns
+      name: coredns
       namespace: kube-system
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
we should start using `v1` instead of `v1beta1`, which will get deprecated soon

Reply:
hmm, in upstream they still on v1beta

Reply:
do you know when it became `v1` ? may be they keep beta for compatibility with old versions?

Reply:
anyway will switch to `v1`

Reply:
it's v1 since 1.8 and we're on 1.9 already in k8scc.v3 so we're safe to update